packages/charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -1,18 +1,16 @@
 import * as React from 'react';
-import { classNamesFunction, getId } from 'office-ui-fabric-react/lib/Utilities';
-import { IDonutChartProps, IDonutChartStyleProps, IDonutChartStyles } from './DonutChart.types';
-import { Pie } from './Pie/Pie';
-import { ILegend, Legends } from '../Legends/index';
 import * as scale from 'd3-scale';
-import { IProcessedStyleSet, IPalette } from 'office-ui-fabric-react/lib/Styling';
-import { IChartDataPoint, IChartProps } from './index';
-import { Callout, DirectionalHint } from 'office-ui-fabric-react/lib/Callout';
 import { FocusZone, FocusZoneDirection } from '@fluentui/react-focus';
-import { ChartHoverCard } from '../../utilities/ChartHoverCard/index';
+import { classNamesFunction, getId } from 'office-ui-fabric-react/lib/Utilities';
+import { Callout, DirectionalHint } from 'office-ui-fabric-react/lib/Callout';
+import { IProcessedStyleSet, IPalette } from 'office-ui-fabric-react/lib/Styling';
+import { Pie } from './Pie/index';
+import { ChartHoverCard, ILegend, Legends } from '../../index';
+import { IChartDataPoint, IChartProps, IDonutChartProps, IDonutChartStyleProps, IDonutChartStyles } from './index';
 
 const getClassNames = classNamesFunction<IDonutChartStyleProps, IDonutChartStyles>();
 
-interface IDonutChartState {
+export interface IDonutChartState {
```

Comment:
why don't we allow just the uniq text being set up the consumer, This could be optional. Why expose all the interface.

Reply:
state interface used to create wrapper of test cases. So exporting and using in test cases.